pkgs/by-name/ma/maltego/package.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
```

```suggestion
  version = "4.6.0";

  src = fetchzip {
```
just a small formatting thing

---

Fixed